packages/core/admin/admin/src/content-manager/pages/EditView/components/FormInputs/DynamicZone/Field.tsx

```diff
@@ -180,28 +180,34 @@ const DynamicZone = ({
     removeFieldRow(name, currentIndex);
   };
 
+  const hasError =
+    error !== undefined ||
+    dynamicDisplayedComponentsLength < min ||
+    dynamicDisplayedComponentsLength > max;
+
   const renderButtonLabel = () => {
     if (addComponentIsOpen) {
       return formatMessage({ id: 'app.utils.close-label', defaultMessage: 'Close' });
     }
 
-    // if (hasError && dynamicZoneError.id?.includes('max')) {
-    //   return formatMessage({
-    //     id: 'components.Input.error.validation.max',
-    //     defaultMessage: 'The value is too high.',
-    //   });
-    // }
-
-    // if (hasError && dynamicZoneError.id?.includes('min')) {
-    //   return formatMessage(
-    //     {
-    //       id: getTranslation(`components.DynamicZone.missing-components`),
-    //       defaultMessage:
-    //         'There {number, plural, =0 {are # missing components} one {is # missing component} other {are # missing components}}',
-    //     },
-    //     { number: missingComponentNumber }
-    //   );
-    // }
+    if (hasError && dynamicDisplayedComponentsLength > max) {
+      return formatMessage({
+        id: getTranslation(`components.DynamicZone.extra-components`),
+        defaultMessage:
+          'There {number, plural, =0 {are # extra components} one {is # extra component} other {are # extra components}}',
```

Comment:
Do you need to pass the number param here too?

Reply:
yes, i do.